assets/src/utils/common_utils.py

```diff
@@ -248,21 +248,25 @@ def convert_to_jpg(self) -> bytes:
         return byte_im
 
     def convert_txt(self) -> bytes:
-        input_text_path = f"files/{self.new_file.id}/{self.new_file.id}.txt"
+        pdf_from_txt = self.convert_to_pdf()
         output_pdf_path = f"files/{self.new_file.id}/{self.new_file.id}.pdf"
+        file_ = BytesIO(pdf_from_txt)
+        self.minio_client.put_object(
+            bucket_name=self.bucket_storage,
+            object_name=output_pdf_path,
+            data=file_,
+            length=len(pdf_from_txt),
+        )
```

Comment:
Are you sure that putting the object shoudn't be done with assets?
If you put it with minio_client directly, assets will not know about it. It means, that UI will not know about it too and will not show it

Reply:
All this preparations is in is_blank_is_created() function. There is a creation of blank record in database and new path for files.